Licensing: update topic for new license mechanism (#8772)
DevExpress/devextreme-documentation: concepts/Common/Licensing/10 Validate Your License Key/30 Next.js Specifics.md

> DevExtreme components function on the client side, which requires a license check during client-side code execution. Create a `ClientComponent` that registers the assigned license key, and run it in the `RootLayout` section before rendering any content.

Reviewer: ```suggestion
DevExtreme components run on the client side, so the license check must also run on the client. Create a `ClientComponent` that registers the assigned license key, and run it in the `ClientComponent` before any content is rendered.
```
